new-client/src/plugins/Edit/EditModel.js

```diff
@@ -345,10 +345,14 @@ class EditModel {
 
     // Make sure we have a name for geometry column. If there are features already,
     // take a look at the first one and get geometry field's name from that first feature.
-    // If there are no features however, default to 'geom'. If we don't then OL will
-    // fallback to its own default geometry field name, which happens to be 'geometry' and not 'geom.
+    // If there are no features however, check if the edit layer has a 'geometryField' property and use this.
+    // this enables users using databases that do not default to 'geom' to set their own standard geometry name.
+    // Lastly, detfault to 'geom'. If we don't then OL will fallback to its own default geometry field name,
```

Comment:
~~detfault~~ => default

Tip: A spell-checker in the code editor simplifies life!